### PR TITLE
Modify UI style

### PR DESCRIPTION
**Pull request by JungoKim**
Add new button of more-asks
Fix bug multiple floating button

### CODE REVIEW

File: client/src/app/components/card-list.jsx [hunk 1]
```diff
@@ -149,12 +172,18 @@ var CardList = React.createClass({
       );
     };
     return (
-      <div className="cardList">
-        {showCard()}
-        {showCard()}
-        {showCard()}
-        {showCard()}
-      </div>
+       <div className="cardList">
```
Comment (bongsok) [inline client/src/app/components/card-list.jsx]: Please align indent

File: client/src/app/components/feed.jsx [hunk 2]
```diff
@@ -41,20 +41,24 @@ var Feed = React.createClass({
...
       <div>
         <div style={containerStyle}>
-          <Tabs style={tabStyle} tabItemContainerStyle={tabItemContainerStyle} inkBarStyle={inkBarStyle} >
-            <Tab label='NEW' >
+          <Tabs style={tabStyle} tabItemContainerStyle={tabItemContainerStyle} inkBarStyle={inkBarStyle}>
+            <Tab label='NEW' style={labelStyle}>
               <NewFeed/>
             </Tab>
-            <Tab label='HOT' >
+            <Tab label='BEST' style={labelStyle}>
```
Comment (bongsok) [inline client/src/app/components/feed.jsx]: Do you want to change the name of this tab?

File: client/src/app/components/master.jsx [hunk 3]
```diff
@@ -147,11 +148,17 @@ class Master extends React.Component {
...
+      logo: {
+        width: 72,
+        height: 18
+      }
     };
 
     var yesOrNoIcon= (
       <EnhancedButton>
-        <span style={styles.span}>Y|N</span>
+         <span style={styles.span}>
```
Comment (bongsok) [inline client/src/app/components/master.jsx]: Please align indent

File: client/src/app/components/feed.jsx (hunk 2 above)
Comment (JungoKim) [inline client/src/app/components/feed.jsx]: yes i do, i think best is clear than hot

**Comment by Myoungsun-Lee** [comment]
우선 1차로 이 UI로 고정해서 UX 전체 플로우를 그리겠습니다.

**Comment by JungoKim** [comment]
I add new patch regarding "Remove MY tab and Change left-nav items"
Please review this.

**Pull request merged**